BridgeSDK/BridgeAPI/SBBUploadManagerInternal.h

```diff
@@ -0,0 +1,50 @@
+//
```

Comment:
it's not defining the category SBBUploadManager(Internal), it's just a "private" header to give unit tests access to some internal stuff.